ingestion/src/metadata/ingestion/source/database/snowflake/metadata.py

```diff
@@ -335,7 +334,16 @@ def query_table_names_and_types(
             or []
         ]
 
-        return regular_tables + external_tables
+        transient_tables = [
+            TableNameAndType(name=table_name, type_=TableType.Transient)
+            for table_name in self.inspector.get_table_names(
+                schema=schema_name,
+                include_temp_tables=self.service_connection.includeTempTables,
```

Comment:
@vanshika18 I think we will need to change this variable to `includeTransientTables` right?

Reply:
yeah you are right! thanks will do this change.